test/gtest/ucp/test_ucp_proto_mock.cc

```diff
@@ -813,3 +866,65 @@ UCS_TEST_P(test_ucp_proto_mock_gpu, cuda_managed_ppln_host_frag,
 
 UCP_INSTANTIATE_TEST_CASE_TLS(test_ucp_proto_mock_gpu, rcx_gpu,
                               "rc_x,cuda,rocm")
+
```

Comment:
i think we have to guard these tests with HAVE_CUDA (and possibly the previous one)